composeml/tests/test_label_transforms/test_sample.py

```diff
@@ -96,3 +98,25 @@ def test_single_target(total_spent):
     match = 'must first select an individual target'
     with pytest.raises(AssertionError, match=match):
         lt.sample(2)
+
+
+def test_sample_per_instance():
```

Comment:
is this test any different from the standard behavior? It might be useful to add another test that verifies that each group is sampled properly, so a group with 4 instances and a group with 2 each have 2 and 1 samples respectively if the fraction is 0.5.

Reply:
Yes, this is different from the standard behavior since the parameter `n` is used. I added another test case for the parameter `frac` as you described.